package.json

```
@@ -29,9 +29,11 @@
     "lint": "run-p -l lint:*",
     "lint:eslint": "eslint --ext .js,.mjs,.cjs,.ts,.mts,.cts ./",
     "lint:md": "prettier --check ./**/*.md",
+    "lint:yml": "prettier --check ./**/*.yml",
```

IMO: It's better to run prettier to supported files in a single execution.
In addition, it's better to run prettier to both `.yml` and `.yaml`.

```suggestion
    "lint:prettier": "prettier --check '**/*.{json,md,yml,yaml}'",
```